### PR TITLE
Added parsing netfilter conntrack statistics from net/stat/nf_conntrack

### PR DESCRIPTION
**Pull request by kozl**
Added parsing of conntrack statistics from procfs for this PR in node_exporter project: https://github.com/prometheus/node_exporter/pull/1155

@discordianfish please review

### CODE REVIEW

File: net_conntrackstat.go [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+	"strconv"
+	"strings"
+
+	"github.com/prometheus/procfs/internal/util"
+)
+
+// A ConntrackStatEntry represents one line from net/stat/nf_conntrack
+// and contains netfilter conntrack statistics at one CPU core
+type ConntrackStatEntry struct {
+	CPU           int
```
Comment (pgier) [inline net_conntrackstat.go]: Is CPU necessary here?  Seems like this can be determined by the location in the slice.

File: net_conntrackstat.go [hunk 2]
```diff
@@ -0,0 +1,154 @@
+// Copyright 2019 The Prometheus Authors
```
Comment (pgier) [inline net_conntrackstat.go]: Should be 2020

File: net_conntrackstat_test.go [hunk 3]
```diff
@@ -0,0 +1,89 @@
+// Copyright 2018 The Prometheus Authors
```
Comment (pgier) [inline net_conntrackstat_test.go]: Should be 2020

**Comment by pgier** [comment]
Fixes #217

File: net_conntrackstat.go (hunk 2 above)
Comment (kozl) [reply net_conntrackstat.go]: Fixed

File: net_conntrackstat_test.go (hunk 3 above)
Comment (kozl) [reply net_conntrackstat_test.go]: Fixed

File: net_conntrackstat.go (hunk 1 above)
Comment (kozl) [reply net_conntrackstat.go]: Yeah, probably this is excessive, removed it.

File: net_conntrackstat.go [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+			return nil, err
+		}
+		entries = append(entries, *conntrackEntry)
+	}
+
+	return entries, nil
+}
+
+// Parses a ConntrackStatEntry from given array of fields
+func parseConntrackStatEntry(fields []string) (*ConntrackStatEntry, error) {
```
Comment (discordianfish) [inline net_conntrackstat.go]: Check for length of fields to avoid a runtime panic.

**Comment by pgier** [comment]
Thanks for the update, looks like the new commit doesn't have a signoff, you can either squash it into the first commit, or do a `git commit --amend --signoff`

**Comment by kozl** [comment]
> Thanks for the update, looks like the new commit doesn't have a signoff, you can either squash it into the first commit, or do a `git commit --amend --signoff`

:ok_hand:

File: net_conntrackstat.go (hunk 4 above)
Comment (kozl) [reply net_conntrackstat.go]: Added

**Pull request merged**